server/models/user.model.js

```diff
@@ -18,7 +18,9 @@ const UserSchema = new Schema(
     resetTokenExpirationDate: { type: Date },
     signUpHash: { type: String },
     signUpHashExpirationDate: { type: Date },
-    surname: { type: String }
+    surname: { type: String },
+    emailNotificationsFrequency: { type: String, default: '' },
+    emailReportSentAt: { type: Date, default: '' }
```

Comment:
Please consider renaming to `lastEmailNotificationSentAt`. Please do not assign empty string as the default - the type is `Date`. As this is not required, not need for default.